### PR TITLE
adding source bash_env to load custom envs should solve #311

### PR DESCRIPTION
**Pull request by brondum**
This enable people to load secrets from envs instead of populating them in CONTEXTS or Projects Envs in CircleCI. Hence the latest CircleCI problems with access to their data.

### CODE REVIEW

**Comment by brondum** [comment]
#311

**Comment by jazanne** [comment]
@brondum thank you for submitting this PR. We will have the team review the change and get back to you.

File: build/package/circleci/orb.yml [hunk 1]
```diff
@@ -115,6 +115,7 @@ jobs:
       - run:
           name: Find flag references
           command: |
+            source $BASH_ENV || true
```
Comment (pkaeding) [inline build/package/circleci/orb.yml:118]: Doesn't CircleCI [already do this](https://circleci.com/docs/env-vars/#parameters-and-bash-environment), automatically?

> In every step, CircleCI uses bash to source BASH_ENV. This means that BASH_ENV is automatically loaded and run, allowing you to use interpolation and share environment variables across run steps.

Comment (brondum) [reply build/package/circleci/orb.yml:118]: Only if the you use `bash` as shell and you specifically asks to use `sh`: 
https://github.com/launchdarkly/ld-find-code-refs/blob/main/build/package/circleci/orb.yml#L111

**Review by jazanne — APPROVED** [review]

**Pull request merged**